### ISSUE TITLE
Getting and Setting Robot DOF Values

### ISSUE THREAD

**Issue by midhun-pm** [AAIR-lab/aair_vrep#1]
**Task** :  : Implement methods to get an set robot DOF values:

Consider how this can be achieved using openrave:
```
from openravepy import *
env = Environment()
env.Load('/usr/local/share/openrave-0.9/data/lab1.env.xml')
env.GetRobots()[0]
robot = env.GetRobots()[0]
**robot.GetDOFValues()**
```
Similarly , you may try out the robot.SetDOFValues() in OpenRave

The task is to create these convenience methods for the V-REV simulator

**Prerequisites**: 
A working OpenRave copy
Study Tutorials Getting started with V-REP
Study about the Remote API in V-REP

All code must reside in python_api/api.py

**Comment by jjbowman2** [comment]
To clarify, are these accessors and mutators already available in OpenRave, but not in V-REP?

**Comment by jjbowman2** [comment]
To clarify, are these accessors and mutators already available in OpenRave, but not in V-REP? (Also my apologies for the close. I mis-clicked)

**Comment by midhun-pm** [comment]
Yeah, these are already present in OpenRave but need to be implemented for
V-REP
It must be implemented generically to work for any robot. Every robot is a
kinematic chain with joints and links so it can be generalized

**Comment by jjbowman2** [comment]
I think the use of this function may be helpful in calculating this. 
![vrep function](https://user-images.githubusercontent.com/22969728/45520291-8544d880-b76d-11e8-8d25-79585719045a.PNG)
The ikGroupHandle can be retrieved using another function. I haven't figured out where the jointcnt and the joint handles will come from given a robot, but I will look into it. The output of the jointValues is what I assume to be our DOF values, correct? (or at least calculated from them)
